base/reflection.jl

```diff
@@ -84,6 +84,12 @@ are also included.
 As a special case, all names defined in `Main` are considered \"public\",
 since it is not idiomatic to explicitly mark names from `Main` as public.
 
+!!! note
+    `sym ∈ names(SomeModule)` does *not* imply `isdefined(SomeModule, sym)`,
+    because the value returned by `names` will contain a symbol marked with `public`
+    (or `export`) in the module, even if that symbol does not correspond to any names
+    defined in the module.
```

Comment:
Some suggestions on the wording:
```suggestion
    `sym ∈ names(SomeModule)` does *not* imply `isdefined(SomeModule, sym)`.
    `names` will return symbols marked with `public` or `export`, even if
    they are not defined in the module.
```
- Put "export" and "public" on an equal footing
- Don't say "because" because the underlying reason for this has more to do with the Julia object model.
- Concision without loss of meaning